cf/test-pages.js

```diff
@@ -0,0 +1,22 @@
+// Add your database url and run this file with
```

Comment:
This file gives some instructions to test the code on pages wrangler (notice the `pages` in the command)